File: _assets/output_gen/Entities/UserAccountRole.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace OB.Data.Entities
{
    public class UserAccountRole
    {
        //--------------------------------------------------------
        [Key]
        public int Id { get; set;}

        public int UserAccountId { get; set;}

        public int RoleId { get; set;}

        public UserAccount UserAccount { get; set;}

        public Role Role { get; set;}

        //--------------------------------------------------------
    }
}

File: _assets/output_gen/jsModels/UserAccountRole.js

//---------------------------------------------------------------------------

export default class UserAccountRole {
  constructor(options) {

    this.Id = null;
    this.UserAccountId = null;
    this.RoleId = null;
    this.UserAccount = null;
    this.Role = null;

    if(options) {
      this.init(options);
    }
  }

  //---------------------------------------------------------------------------
  init(options) {

    this.Id = options.Id || this.Id;
    this.UserAccountId = options.UserAccountId || this.UserAccountId;
    this.RoleId = options.RoleId || this.RoleId;
    this.UserAccount = options.UserAccount || this.UserAccount;
    this.Role = options.Role || this.Role;

  }  
  //---------------------------------------------------------------------------  
}